tests/test_hyperframe.py

```diff
@@ -0,0 +1,52 @@
+import gzip
+import json
+
+import pandas as pd
+from bs4 import BeautifulSoup
+from pandas._testing import assert_frame_equal
+
+from hyperframe.hyperpattern import HyperPatternHunter
+
+EXPECTED_HYPERFRAMES_PATH = "tests/fixtures/beauhurst_expected_hyperframe_fixture.json"
+BEAUHURST_HTML_PATH = "tests/fixtures/beauhurst_scrapedpage_html.gz"
+
+
+def get_expected_hyperframes() -> list[pd.DataFrame]:
```

Comment:
probably good to use `pytest.fixture` here

Reply:
https://github.com/sam-lawrence/HyperFrame/pull/8/commits/cd5de52049305d5a0422f4e77ed05534249ca13f